Automattic/jetpack
# Gutenpack: Expose data for blocks

Closes #10243 (supersedes)
Part of #10198

Calypso (block) PR in progress: https://github.com/Automattic/wp-calypso/pull/27875

This is the simplest way of reusing an existing data set to expose some required data for blocks to access.

#### Changes proposed in this Pull Request:

* Exposes `Jetpack_Initial_State` global. This is the same data used by the React-based dashboard/settings views

#### Testing instructions:

* Load the Gutenberg editor with Jetpack blocks available
* Observe the `Jetpack_Initial_State` global is available in the console
